LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 AknEp
```

Comment:
This repository should include the copyright notice from Pixi: https://github.com/pixijs/pixi.js/blob/dev/LICENSE

It should be copyright by myself, Mat, or both of us.